Makefile

```diff
@@ -0,0 +1,40 @@
+HOST = localhost:4000
+NPM_BIN = ./node_modules/.bin
+export PATH := $(NPM_BIN):$(PATH)
+
+# Install dependencies
+install:
+	@echo "Installing Pa11y..."
+	@npm install pa11y@^3
+	@echo "Installing GitHub Pages gem..."
+	@gem install github-pages
+
+# Build the site
+build:
```

Comment:
They're not really for Circle but for running locally. You can use the `make serve` as an example to run the site locally without typing `jekyll serve --watch`. It comes down to developer preference I guess, and I think it is quite useful to keep.